Result.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'Result'
-  s.version      = '3.2.1'
+  s.version      = '4.0.0-alpha.1'
```

Comment:
Swift Package Manager in Swift 3.1 would not work with pre-release tag.
https://bugs.swift.org/browse/SR-1039